tests/common.go

```diff
@@ -95,6 +99,114 @@ func GetDefaultNetwork(networks *compute.NetworksClient, gcpProject, networkName
 	return network
 }
 
+func CreateIPAddress(addresses *compute.AddressesClient, gcpProject, name string) *computepb.Address {
```

Comment:
I was wondering why this and other similar functions don't receive the `context` from the test rather than creating it inside the function.

Reply:
The `context` in the tests are (unless specifically testing for canceled context) always just `context.Background()` and the `ctx` var is there mostly for convenience than anything else. So here I just didn't want to pass an extra parameter to the functions. Some of them already have a rather large list of params.